requirements/test.txt

```diff
@@ -4,43 +4,43 @@
 #
 #    make upgrade
 #
-astroid==2.12.5
+astroid==2.12.10
     # via pylint
 attrs==22.1.0
     # via
     #   jschema-to-python
     #   jsonschema
     #   pytest
     #   sarif-om
-aws-sam-translator==1.50.0
+aws-sam-translator==1.51.0
     # via cfn-lint
 aws-xray-sdk==2.10.0
     # via moto
 boto==2.49.0
     # via
     #   -r requirements/base.txt
     #   google-compute-engine
-boto3==1.24.63
+boto3==1.24.84
     # via
     #   -r requirements/base.txt
     #   aws-sam-translator
     #   moto
-botocore==1.27.63
+botocore==1.27.84
     # via
     #   -r requirements/base.txt
     #   aws-xray-sdk
     #   boto3
     #   moto
     #   s3transfer
-certifi==2022.6.15
+certifi==2022.9.24
     # via requests
 cffi==1.15.1
     # via cryptography
-cfn-lint==0.63.2
+cfn-lint==0.66.0
     # via moto
 charset-normalizer==2.1.1
     # via requests
-cryptography==37.0.4
+cryptography==38.0.1
```

Comment:
Nothing backward incompatible in this release. Package upgrade looks good. 
https://cryptography.io/en/latest/changelog/